citizenvoice/citizenvoice/settings.py

```diff
@@ -95,14 +97,21 @@
 # Database
 # https://docs.djangoproject.com/en/4.0/ref/settings/#databases
 
+# DATABASES = {
```

Comment:
Keep the configuration for the Postgres database. Sqlite3 doesn't support spatial types, and the spatialite extension gives me many errors when migrating.

Reply:
As far as I know, we finally decided on our meeting on the 15th of August to only use SQLite for now. So should I still include it but leave it commented out?